lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -122,8 +122,9 @@ exports.create = (Dep, range, param, expr, options, contextOptions, parser) => {
 					value = value + "`";
 				} else if (
 					part.expression &&
-					part.expression.type === "TemplateElement" &&
-					part.expression.value.raw === value
+					part.expression.type === "TemplateLiteral" &&
```

Comment:
could you please check this

Reply:
revert that and change type of `expression` to `estree.Node`